backend/src/main/scala/cromwell/backend/package.scala

```diff
@@ -0,0 +1,35 @@
+package cromwell
+
+import com.typesafe.config.Config
+import cromwell.core.{WorkflowId, WorkflowOptions}
+import wdl4s._
+import wdl4s.values.WdlValue
+
+package object backend {
```

Comment:
I'm open to the idea... but if it's only to define the interface that Backend is presenting to the world, then I don't mind leaving it in backend. 

OTOH as I say, if there's a bunch of other stuff to throw in there, I'm open to the idea